Core/CMakeLists.txt

```diff
@@ -254,13 +258,13 @@ set(ELASTIX_LIBRARIES elastix_lib transformix_lib PARENT_SCOPE)
 #---------------------------------------------------------------------
 # Define the install directory for elastix and transformix.
 
-if(NOT WIN32)
+if(NOT WIN32 AND NOT WASI AND NOT EMSCRIPTEN)
   # Tell the executables where to find the required .so files.
   set_target_properties(elastix_exe transformix_exe
     PROPERTIES INSTALL_RPATH "${CMAKE_INSTALL_PREFIX}/lib:${ITK_DIR}")
 endif()
 
-if(NOT ELASTIX_NO_INSTALL_EXECUTABLES)
+if(NOT ELASTIX_NO_INSTALL_EXECUTABLES AND NOT WASI AND NOT EMSCRIPTEN)
```

Comment:
I guess this could instead be achieved at the WASI side, by setting `ELASTIX_NO_INSTALL_EXECUTABLES`. Right?

Reply:
Yes, I can do that.